From 06f863aee2203bfb1f8fadc0577ffb72094679c7 Mon Sep 17 00:00:00 2001
From: Josh Wilsdon <josh@wilsdon.ca>
Date: Wed, 3 Oct 2018 11:32:59 -0700
Subject: [PATCH] TRITON-822 DAPI should have a filter which can reliably fail
 a provision Reviewed by: Marsell Kukuljevic <marsell@joyent.com> Approved by:
 Marsell Kukuljevic <marsell@joyent.com>

---
 package.json                  | 4 ++--
 sapi_manifests/cnapi/template | 1 +
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/package.json b/package.json
index 2f079ba..bea12a2 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.12.2",
+  "version": "1.13.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -9,7 +9,7 @@
     "assert-plus": "1.0.0",
     "async": "1.5.2",
     "bunyan": "1.8.5",
-    "dapi": "git+https://github.com/joyent/sdc-designation.git#45a8d7c367300d3003838a4e0241915cf89fe3f9",
+    "dapi": "git+https://github.com/joyent/sdc-designation.git#99c56f2b7eef4bac298807cf43016a4c971896f9",
     "deep-equal": "1.0.1",
     "dox": "0.9.0",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
diff --git a/sapi_manifests/cnapi/template b/sapi_manifests/cnapi/template
index 48d749b..d365a15 100644
--- a/sapi_manifests/cnapi/template
+++ b/sapi_manifests/cnapi/template
@@ -81,6 +81,7 @@
 			"pipe", "hard-filter-setup",
 			        "hard-filter-running",
 			        "hard-filter-invalid-servers",
+			        "hard-filter-force-failure",
 			        "hard-filter-volumes-from",
 			        "hard-filter-reserved",
 			        "hard-filter-min-free-disk",
-- 
2.21.0

